content/reference-rippled.md

```diff
@@ -7571,6 +7750,247 @@ In addition to the standard Offer fields, the following fields may be included i
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+The request includes the following parameters:
+
+| Field | Type | Description |
+|-------|------|-------------|
+| `amount` | String | The amount of [XRP, in drops][], the provided `signature` authorizes. |
+| `channel_id` | String | The Channel ID of the channel that provides the XRP. This is a 64-character hexadecimal string. |
+| `public_key` | String | The public key of the channel and the key pair that was used to create the signature, in base58 format. (One way to get the public key in base58 format is from the [`wallet_propose` command](#wallet-propose).) |
+| `signature` | String | The signature to verify, in hexadecimal. |
+
+#### Response Format ####
+
+An example of a successful response:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*WebSocket*
+
+```
+{
+    "id": 1,
+    "status": "success",
+    "type": "response",
+    "result": {
+        "signature_verified":true
+    }
+}
+```
+
+*JSON-RPC*
+
+```
+200 OK
+
+{
+    "result": {
+        "signature_verified":true,
+        "status":"success"
+    }
+}
+```
+
+*Commandline*
+
+```
+{
+    "result": {
+        "signature_verified":true,
+        "status":"success"
+    }
+}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+The response follows the [standard format](#response-formatting), with a successful result containing the following fields:
+
+| Field | Type | Description |
+|-------|------|-------------|
+| `signature_verified` | Boolean | If `true`, the signature is valid for the stated amount, channel, and public key. |
+
+**Caution:** This does not indicate check that the channel has sufficient XRP allocated to it. Before considering a claim valid, you should look up the channel in the latest validated ledger and confirm that the channel is open and its `amount` value is equal or greater than the `amount` of the claim. To do so, use the [`account_channels` method](#account-channels).
+
+#### Possible Errors ####
+
+* Any of the [universal error types](#universal-errors).
+* `invalidParams` - One or more fields are specified incorrectly, or one or more required fields are missing.
+* `publicMalformed` - The value specified in the `public_key` field of the request was not a valid public key in the correct format. Public keys should be 33 bytes, represented in base58. The base58 representation should start with letter `a`.
+* `channelMalformed` - The value specified in the `channel_id` field of the reqeuest was not a valid Channel ID. The Channel ID should be a 256-bit (64-character) hexadecimal string.
+* `channelAmtMalformed` - The value specified in the `amount` field was not a valid XRP amount. See [Specifying Currency Amounts](#specifying-currency-amounts) for details.
```

Comment:
Present tense and 'should' >> 'must':
> * `channelMalformed` - The value specified in the `channel_id` field of the reqeuest is not a valid Channel ID. The Channel ID must be a 256-bit (64-character) hexadecimal string.
 * `channelAmtMalformed` - The value specified in the `amount` field is not a valid XRP amount. See [Specifying Currency Amounts](#specifying-currency-amounts) for details.